blueprints/app/files/package.json

```diff
@@ -41,11 +41,14 @@
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^4.6.0",
     "ember-resolver": "^7.0.0",
-    "ember-source": "~3.17.0-beta.1<% if (welcome) { %>",
+    "ember-source": "~3.17.0-beta.1",
+    "ember-template-lint": "^2.0.0<% if (welcome) { %>",
     "ember-welcome-page": "^4.0.0<% } %>",
+    "eslint": "^6.8.0",
     "eslint-plugin-ember": "^7.7.2",
     "eslint-plugin-node": "^11.0.0",
     "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
To make it easier for others to find, `npm-run-all` is a very popular (~ 4M downloads/month) package made to do _exactly_ the thing we are trying to do (with some neat features like glob support).

https://www.npmjs.com/package/npm-run-all